week09_policy_II/ppo.ipynb

```diff
@@ -472,7 +482,6 @@
     "\n",
     "* [Coefficient of Determination](https://en.wikipedia.org/wiki/Coefficient_of_determination) between \n",
     "value targets and value predictions\n",
-    "* Entropy of the policy $\\pi$\n",
```

Comment:
Why was this line removed?

Reply:
In this task we do not use entropy component loss (it was used in week06, and think this paragraph is copy-paste typo).